### PR TITLE
add more symbolic tests

### PR DESCRIPTION
**Pull request by daejunpark**
@cos please review. just added more tests.

### CODE REVIEW

File: API/src/main/java/org/kframework/Kapi.java [hunk 1]
```diff
@@ -202,7 +206,95 @@ public static void kprint(CompiledDefinition compiledDef, RewriterResult result)
      * compiledDef1: for symbolic execution
      * compiledDef2: for symbolic execution
      */
-    public static void kequiv(CompiledDefinition compiledDef0, CompiledDefinition compiledDef1, CompiledDefinition compiledDef2, String proofFile, String prelude) {
+    public static void kequiv(CompiledDefinition compiledDef0, CompiledDefinition compiledDef1, CompiledDefinition compiledDef2, String proofFile1, String proofFile2, String prelude) {
```
Comment (cos) [inline API/src/main/java/org/kframework/Kapi.java:209]: rename  var names to match their function, e.g., `compiledDef0` => `parsingSpecDef`

File: API/src/main/java/org/kframework/Kapi.java [hunk 2]
```diff
@@ -202,7 +206,95 @@ public static void kprint(CompiledDefinition compiledDef, RewriterResult result)
...
+        assert ((KItem) e.equalities().get(0).leftHandSide()).kLabel() instanceof KLabelConstant;
+        assert ((KLabelConstant) ((KItem) e.equalities().get(0).leftHandSide()).kLabel()).label().equals("vars");
+        assert ((KItem) e.equalities().get(0).leftHandSide()).kList() instanceof KList;
+        assert ((KList) ((KItem) e.equalities().get(0).leftHandSide()).kList()).getContents().size() == 1;
+        assert ((KList) ((KItem) e.equalities().get(0).leftHandSide()).kList()).getContents().get(0) instanceof BuiltinList;
+
+        return ((BuiltinList) ((KList) ((KItem) e.equalities().get(0).leftHandSide()).kList()).getContents().get(0)).children;
+    }
+
+    // TODO: better name
```
Comment (cos) [inline API/src/main/java/org/kframework/Kapi.java:271]: rename

File: API/src/main/java/org/kframework/Kapi.java [hunk 3]
```diff
@@ -253,113 +345,113 @@ public static void kequiv(CompiledDefinition compiledDef0, CompiledDefinition co
...
+            trusted.add(startRule.containsAttribute(Attribute.TRUSTED_KEY));
         }
 
-        boolean result = EquivChecker.equiv(startSyncNodes1, startSyncNodes2, targetSyncNodes1, targetSyncNodes2, ensures, trusted, rewriter1, rewriter2);
-        System.out.println(result);
+        return new Info(startSyncNodes, targetSyncNodes, startEnsures, targetEnsures, trusted, rewriter);
+    }
 
-        return;
+    // TODO: better name
```
Comment (cos) [inline API/src/main/java/org/kframework/Kapi.java:421]: rename

File: kernel/src/main/java/org/kframework/keq/KeqOptions.java [hunk 4]
```diff
@@ -30,22 +28,27 @@
     @ParametersDelegate
     public DefinitionLoadingOptions definitionLoading = new DefinitionLoadingOptions();
 
-    @Parameter(names="--def0", description="K definition of aggregated configuration.")
+    @Parameter(names="--def0", description="K kompiled definition of common language.")
```
Comment (cos) [inline kernel/src/main/java/org/kframework/keq/KeqOptions.java:31]: the parameter names could be better

**Review by cos — CHANGES_REQUESTED** [review]
looks good overall; I mostly just mirrored your own TODOs

**Comment by daejunpark** [comment]
Close deferred PR. Labeled with `deferred` for later search purpose.

**Pull request closed without merging**